portfolio/src/main/webapp/script.js

```diff
@@ -83,4 +83,32 @@ function createComment(text) {
   const liComment = document.createElement('li');
   liComment.innerText = text;
   return liComment;
+}
+
+google.charts.load('current', {'packages':['geochart'],
+  'mapsApiKey': 'AIzaSyBWdnwt9lVQ2rh-46MMfEWBDH9E5UVun7g'});
```

Comment:
For readability here, it might be better to format this like:

```
google.charts.load('current', {
  'packages': ['geochart'],
  'mapsApiKey': 'AIzaSyBWdnwt9lVQ2rh-46MMfEWBDH9E5UVun7g'
});
```